dcp_prototype/backend/ledger/code/common/ledger_orm.py

```diff
@@ -24,161 +24,232 @@ def session(self, **kwargs):
+    created_at = Column(DateTime(True), nullable=False, server_default=text("now()"))
+    updated_at = Column(DateTime(True), nullable=False, server_default=text("now()"))
 
 
 class BiosamplePrep(Base):
     __tablename__ = "biosample_prep"
-    id = Column(String(), primary_key=True)
-    category = Column(String(), nullable=True)
-    organ_ontology = Column(String(), nullable=True)
-    developmental_stage = Column(String(), nullable=True)
-    disease_ontology = Column(String(), nullable=True)
-    created_at = Column(DateTime, default=datetime.utcnow, nullable=False)
-    updated_at = Column(
-        DateTime, default=datetime.utcnow, nullable=False, onupdate=datetime.utcnow
-    )
 
-    library_prep_protocols = relationship(
-        "LibraryPrepProtocol",
-        order_by=LibraryPrepProtocol.id,
-        back_populates="biosample_prep",
-    )
+    id = Column(String, primary_key=True)
+    category = Column(String)
+    organ_ontology = Column(String)
+    developmental_stage = Column(String)
+    disease_ontology = Column(String)
+    created_at = Column(DateTime(True), nullable=False, server_default=text("now()"))
+    updated_at = Column(DateTime(True), nullable=False, server_default=text("now()"))
 
 
 class Contributor(Base):
     __tablename__ = "contributor"
-    id = Column(String(), primary_key=True)
-    name = Column(String(), nullable=True)
-    email = Column(String(), nullable=True)
-    phone_number = Column(String(), nullable=True)
-    corresponding_contributor = Column(String(), nullable=True)
-    lab = Column(String(), nullable=True)
-    street_address = Column(String(), nullable=True)
-    country = Column(String(), nullable=True)
-    contributor_role_ontology = Column(String(), nullable=True)
-    orcid_id = Column(String(), nullable=True)
-    created_at = Column(DateTime, default=datetime.utcnow, nullable=False)
-    updated_at = Column(
-        DateTime, default=datetime.utcnow, nullable=False, onupdate=datetime.utcnow
-    )
 
-    project_id = Column(String(), ForeignKey("project.id"), nullable=False)
-    project = relationship("Project", back_populates="contributors")
+    id = Column(String, primary_key=True)
+    name = Column(String)
+    email = Column(String)
+    phone_number = Column(String)
+    corresponding_contributor = Column(String)
+    lab = Column(String)
+    street_address = Column(String)
+    country = Column(String)
+    contributor_role_ontology = Column(String)
+    orcid_id = Column(String)
+    created_at = Column(DateTime(True), nullable=False, server_default=text("now()"))
+    updated_at = Column(DateTime(True), nullable=False, server_default=text("now()"))
+
+
+class ExpressionFile(Base):
+    __tablename__ = "expression_file"
+
+    id = Column(String, primary_key=True)
+    filename = Column(String)
+    file_format = Column(String)
+    file_size = Column(String)
+    created_at = Column(DateTime(True), nullable=False, server_default=text("now()"))
```

Comment:
Probably also needs an s3_uri. 
Also are you assuming that `quantification_process_id` is the same as a `quantification_protocol_id`?

Reply:
re: Last comment: I see `quantification_process_id` in the list of fields of the ExpressionFile table in the diagram

Reply:
s3 uri added. Yes, I assumed quantification_process_id is the same as quantification_protocol_id. Is that a unique field? (seems odd, sounds like a fk with no matching table)

Reply:
I'm checking with Jason now.